app/lib/features/pins/actions/report_pin_action.dart

```diff
@@ -0,0 +1,17 @@
+// report pin dialog
+import 'package:acter/common/actions/report_content.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+void showReportDialog(BuildContext context, ActerPin pin) {
```

Comment:
I'm thinking of a way of having generic type which encompasses any ```ActerModel``` which can be passed down here and avoid creating redact/report dialogs for each type...

Reply:
Same as mentioned on above comment.